tests/CI/buildAsterisk.sh

```diff
@@ -145,6 +145,7 @@ if [ $NO_MENUSELECT -eq 0 ] ; then
 	runner menuselect/menuselect `gen_cats enable $cat_enables` menuselect.makeopts
 
 	mod_disables="res_digium_phone"
+	mod_disables="res_pjsip_config_sangoma"
```

Comment:
This file is no longer used. Nothing in the CI directory is anymore.  It's on my list to remove.

You'll want to look at the asterisk-ci-actions repo in scripts.